packages/manager-react-components/src/hooks/services/services.type.ts

```diff
@@ -55,6 +55,21 @@ export type ResourceStatus =
   | 'toDelete'
   | 'toSuspend';
 
+export type Renew = {
+  current: CurrentRenew;
+  capacities: RenewCapacities;
+};
+
+export type CurrentRenew = {
+  mode: RenewMode | null;
+  nextDate: string | null;
+  period: string;
+};
+
+export type RenewCapacities = {
+  mode: RenewMode[];
+};
+
```

Comment:
Problem of rebase, the services has fixed on the develop branch with different use